src/formatDate.js

```diff
@@ -1,14 +1,38 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.splice(3, 1);
+  const newSeparator = toFormat.splice(3, 1);
```

Comment:
Similar to the previous line, `splice` is incorrectly used here. Use `split` to handle string separation.